src/common/options/osd.yaml.in

```diff
@@ -484,6 +484,28 @@ options:
     stats (inc. scrub/block duration) every this many seconds.
   default: 120
   with_legacy: false
+# when replicas are slow to respond to scrub resource reservations
+- name: osd_scrub_slow_reservation_response
+  type: int
```

Comment:
uint? Same for timeout

Reply:
I prefer 'int':
- following the type used for the scrub settings above it;
- the general pref for signed numbers for all but bitwise operations.

Reply:
type: millisecs/secs will fit.

Reply:
? I don't see this type in  osd.yaml.in ?
It would be great to have a chrono related type that translates directly to std::chrono types. I don't think
we have it yet

Reply:
Grep for millisecs in global.yaml.in

Reply:
But not is the OSD one. I'll have to make sure it works there.

Reply:
Fixed. Let's hope it works...